### PR TITLE
155 set up python flask backend for angular

### PR DESCRIPTION
**Pull request by moltubaer**

### CODE REVIEW

File: sip-viewer/src/app/app.component.ts [hunk 1]
```diff
@@ -45,6 +50,13 @@ import { RerouteService } from './reroute.service';
     MatIconModule,
     HomeComponent,
     MatButtonModule,
+    MatFormFieldModule,
```
Comment (samer-12) [inline sip-viewer/src/app/app.component.ts:53]: Unused imports?

File: sip-viewer/src/app/file-upload/file-upload.component.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ import { NgIf } from '@angular/common';
 })
 export class FileUploadComponent {
   fileName = '';
-  constructor(private dataService: DataService) {}
+  constructor(private dataService: DataService, private http: HttpClient) {}
```
Comment (samer-12) [inline sip-viewer/src/app/file-upload/file-upload.component.ts:18]: Unused HttpClient

File: sip-viewer/src/app/upload-portal/upload-portal.component.ts [hunk 3]
```diff
@@ -42,15 +49,68 @@ export class UploadPortalComponent {
...
+        a.href = url;
+        a.download = filename;
+        document.body.appendChild(a);
+        a.click();
+        window.URL.revokeObjectURL(url);
+      });
     }
   }
 
+  // ---------------------------------------------------------------------
```
Comment (hellesvaasand) [inline sip-viewer/src/app/upload-portal/upload-portal.component.ts:113]: Remove line

**Review by hellesvaasand — APPROVED** [review]
nice

**Review by samer-12 — APPROVED** [review]

**Pull request merged**